packages/cyberstorm/src/components/componentStyles/MetaItem.module.css

```diff
@@ -0,0 +1,39 @@
+/* Common MetaItem styles */
+.metaItem {
+  display: flex;
+  flex: none;
+  flex-direction: row;
+  gap: var(--size-xs);
+  align-items: center;
+  justify-content: center;
+
+  padding: var(--size-xs) var(--size-xs);
+  color: var(--text-color);
+  background-color: var(--bg-color);
+
+  --text-color: var(--color-default-purple--8);
+  --bg-color: transparent;
+}
+
+/* Styles for MetaItem's text */
+.metaItemLabel {
+  font-weight: 700;
+  font-size: var(--text-xs);
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  line-height: var(--text-m);
+}
+
+/* Styles for MetaItem's icon */
+.metaItemIcon {
+  font-size: var(--text-xs);
+}
+
+/* Different MetaItem color schemes */
+.metaItem__default {
+  --text-color: var(--color-default-purple--8);
+}
+
+.metaItem__green {
```

Comment:
Not sure if this is a thing?

Reply:
Used as an example, not in the designs. Added a comment in the code about this.